paddle/operators/sequence_concat_op.cc

```diff
@@ -68,11 +68,12 @@ class SequenceConcatOpMaker : public framework::OpProtoAndCheckerMaker {
                  "The level should be less than the level number of inputs.")
         .SetDefault(0);
     AddComment(R"DOC(
-Sequence Concat operator
+Sequence Concat operator.
```

Comment:
Can you change this to "Sequence Concat Operator"? We now follows a all-cap format for operator title.

Reply:
Done. Thanks for the update.